test/linter/test_swift_appleswiftformat.vader

```diff
@@ -0,0 +1,37 @@
+Before:
+  call ale#assert#SetUpLinterTest('swift', 'appleswiftformat')
+
+After:
+  call ale#assert#TearDownLinterTest()
+
+Execute(Should use default command when use_swiftpm is not set):
+  call ale#test#SetFilename('../test-files/swift/non-swift-package-project/src/folder/dummy.swift')
+
+  let g:ale_swift_appleswiftformat_executable = 'swift-format'
+  let g:ale_swift_appleswiftformat_use_swiftpm = 0
+
+  AssertLinter 'swift-format', ale#Escape('swift-format') . ' lint %t'
+
+" Execute(Should use default command and available configuration when use_swiftpm is not set):
+"   call ale#test#SetDirectory('/testplugin/test/test-files/swift/swift-package-project-with-config')
+"   call ale#test#SetFilename('src/folder/dummy.swift')
+"
+"   let g:ale_swift_appleswiftformat_executable = 'swift-format'
+"   let g:ale_swift_appleswiftformat_use_swiftpm = 0
+"
+"   AssertLinter 'swift-format', ale#Escape('swift-format') . ' lint %t ' . '--configuration .swift-format'
```

Comment:
Use absolute path for .swift-format file. Ensure to use g:dir variable is you set a directory with SetDirectory().

```
AssertLinter 'swift-format', ale#Escape('swift-format') . ' lint %t ' . '--configuration ' . g:dir . '/.swift-format'
```